addons/udes_priorities/models/priorities.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api, _
+
+PROTECTED_FIELDS = {
+    "reference",
+    "sequence",
+    "picking_type_ids",
+    "group_ids",
+    "active",
+}
+
+
+class UdesPriority(models.Model):
+    _name = "udes_priorities.priority"
```

Comment:
Class is about "priority" and file is called "priorities.py", we should probably rename the file.

Reply:
done